.github/workflows/publish_on_tag.yml

```diff
@@ -10,6 +10,19 @@ jobs:
       - uses: actions/checkout@master
       - id: release_version
         run: echo ::set-output name=value::${GITHUB_REF:11}
+      - name: update pushed tag
+        env:
+          RELEASE_VERSION: ${{ steps.release_version.outputs.value}}
+          OWNER: ${{ github.repository_owner }}
+        run: |
+          sed -i -e "s#: .*/hcloud-csi-driver:latest#: $OWNER/hcloud-csi-driver:$RELEASE_VERSION#" deploy/kubernetes/hcloud-csi.yml
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add .
+          git commit -m "Update image tags in hcloud-csi.yml to $RELEASE_VERSION"
+          git tag -d v$RELEASE_VERSION
```

Comment:
iirc the `v` is to much here. The $RELEASE_VERSION is always the tag, and that is something like `v1.5.0`. So adding `v` here and below is not needed.